ykrt/src/trace/swt/mod.rs

```diff
@@ -99,10 +98,8 @@ impl TraceRecorder for SWTTraceRecorder {
                     }
                 });
 
-                aot_blocks = tb
-                    .borrow()
-                    .iter()
-                    .map(|tb| match fnames.borrow_mut().get(&tb.function_index) {
+                aot_blocks.extend(tb.borrow().iter().map(|tb| {
```

Comment:
This looks like a semantic change?

Reply:
Because of `let mut aot_blocks: Vec<TraceAction> = vec![]` above, it's semantically equivalent. [Or at least I think it is!]

Reply:
It has to be defined earlier so that it can be referenced in the `with` closure. There are probably nicer ways of doing this, but this is the only short-term fix I can see.

Reply:
ah right.